listView.js

```diff
@@ -13,6 +14,15 @@ const RENDER_AHEAD_DISTANCE = 1000000;
 
 class EnhancedListView extends Component {
   static DataSource = ListView.DataSource;
+  static propTypes = {
+    ...ListView.propTypes,
+    onSectionChanged: PropTypes.func
+  };
+  static defaultProps = {
+    // arbitrarily large distance to pre-render all sections for measurements
+    scrollRenderAheadDistance: 1000000,
```

Comment:
Are we sure we want to do this?

It seems like this will force a rendering of our full list, negating any performance optimizations of view recycling.